packages/ui/src/ui/pages/navigation/modals/CreateACOModal.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import cn from 'bem-cn-lite';
+
+import {DialogError, DialogField, FormApi, YTDFDialog} from '../../../components/Dialog/Dialog';
+import {useDispatch, useSelector} from 'react-redux';
+import {getCreateACOModalState} from '../../../store/selectors/navigation/modals/create-aco-modal';
+import {closeCreateACOModal, createACO} from '../../../store/actions/navigation/modals/create-aco';
+
+const block = cn('table-sort-modal');
+
+type CreateACOFormState = {
+    namespace: string;
+    name: string;
+    path: string;
+};
+
+export default function CreateACOModal() {
+    const [error, setError] = React.useState<any>();
+    const {visible, path, namespace} = useSelector(getCreateACOModalState);
+
+    const dispatch = useDispatch();
```

Comment:
To use the dialog from queries page it is better to break the connection with store, i.e. it should not use dispatch/actions/selectors.

Reply:
yt-api is not ready yet